README.md

```diff
@@ -168,31 +169,37 @@ CAIP-29: `{chainNamespace}:{chainReference}/erc1155:{contractAddress}/{tokenId}`
 **DID->CAIP**
 
 ```
-const caip = did.substr(8).replace(/_/g, '/').replace(/\./g, ':')
+const caip = did.substr(8).replace(/_/g, '/')
 ```
 
 **CAIP->DID**
 
 ```
-const did = `did:nft:${caip.replace(/\//g, '_').replace(/:/g, '.')}`
+const did = `did:nft:${caip.replace(/\//g, '_')
 ```
 
 There are helpers that help you with the conversion:
 
 ```typescript
-import { caipToDid, didToCaip } from 'nft-did-resolver'
+import { caipToDid, didToCaip, createNftDidUrl } from 'nft-did-resolver'
 import { AssetId } from 'caip'
 
 // CAIP -> DID URL
 const assetId = new AssetId(
-  AssetId.parse('eip155:1/erc721:0x1234567891234567891234567891234596351156/0x1')
+  AssetId.parse('eip155:1/erc721:0x1234567891234567891234567891234596351156/1')
 )
-const didUrl = caipToDid(assetId) // did:nft:eip155.1_erc721.0x1234567891234567891234567891234596351156_0x1
-const didUrlWithTimestamp = caipToDid(assetId, 1628529680) // did:nft:eip155.1_erc721.0x1234567891234567891234567891234596351156_0x1?versionTime=2021-08-09T17:21:20Z
+const didUrl = caipToDid(assetId) // did:nft:eip155:1_erc721:0x1234567891234567891234567891234596351156_1
+const didUrlWithTimestamp = caipToDid(assetId, 1628529680) // did:nft:eip155:1_erc721:0x1234567891234567891234567891234596351156_1?versionTime=2021-08-09T17:21:20Z
+// If you do not use `caip` library:
+const fromRawParams = createNftDidUrl({
+  chainId: 'eip155:1',
+  namespace: 'erc721:0x1234567891234567891234567891234596351156',
+  tokenId: '1',
+})
```

Comment:
Can we suggest `createNftDidUrl` as the main way of doing it and only document the other way as an option. Right now it's kind of confusing to show both in the same snippet.

Reply:
Done.

Reply:
OK by you?